requirements.txt

```diff
@@ -1,5 +1,6 @@
-gevent>=1.1.0; python_version < "3.8"
-gevent>=1.5.0; python_version >= "3.8"
+gevent==1.1.0; python_version < "3.0"
```

Comment:
This line looks odd, ZeroNet doesn't support Python 2 anymore so it looks like this can be safely removed.